app/controllers/api/v2/tag_group_adapter_types_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # Provides a JSON API controller for TagGroup::AdapterType
+    # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
+    class TagGroupAdapterTypesController < JSONAPI::ResourceController
+      # By default JSONAPI::ResourceController provides most the standard
+      # behaviour, and in many cases this file may be left empty.
+    end
+  end
+end
```

Comment:
TagGroupAdapterType is used for filtering TagGroups by adapter type name. A TagGroup has an association with TagGroupAdapterType using adapter_type_id field.

Reply:
Not sure if you're asking for any changes here.  The documentation for these resources is here: https://github.com/sanger/sequencescape/blob/Y24-190-api-v2-documentation/app/resources/api/v2/tag_group_adapter_type_resource.rb

This PR doesn't intend to write specific documentation; only to set up a framework for it to be updated by people who know more about the resources.